deps/chakrashim/core/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+## Contributing Code
```

Comment:
Is there any need for these files here? 
Why isn't ChakraCore brought as a dependency?